lib/store.js

```
@@ -0,0 +1,67 @@
+var shortid = require('shortid');
+const fs = require('fs');
+const path = require('path');
+
+module.exports = class Store {
+    constructor(dbPath) {
+        this.dbPath = dbPath;
+    }
+
+    save(item, callback) {
+        item._id = shortid.generate();
+        const storableItem = JSON.stringify(item);
+        fs.writeFile(path.join(this.dbPath, `${item._id}.json`), storableItem, err => {
+            if(err) {
+                callback(err);
+            }
+            else {
+                callback(null, item);
+            }
+        });
+    }
+
+    get(_id, callback) {
+        fs.readFile(path.join(this.dbPath, `${_id}.json`), (err, file) => {
+            if(err) {
```

You should throw an error sometimes. You just want to return `callback(null, null)` on the `ENOENT` error:

```
if(err && err.cod === 'ENOENT') callback(null, null)
else if(err) callback(err)
else callback(null, JSON.parse(file))
```

---

Fixed